src/sync/outgoingSyncUtils.js

```diff
@@ -118,10 +116,10 @@ function generateSyncData(settings, recordType, record) {
         ID: record.id,
         date_entered: getDateString(record.entryDate),
         user_ID: record.enteredById,
-        name_ID: settings.get(THIS_STORE_NAME_ID),
+        name_ID: record.otherStoreName.id,  // TODO: check if there is a case when outgoing req might not have otherStoreName
```

Comment:
I can't think of a case, but will leave this to adjust last (in case there is a case)